ZuluContent/Mobiles/BaseCreature/BaseCreature.cs

```diff
@@ -869,15 +869,7 @@ public virtual void OnDamagedBySpell(Mobile from)
                 CheckDistracted(from);
             }
         }
-
-        public virtual void OnHarmfulSpell(Mobile from)
```

Comment:
OnHarmfulSpell not needed anymore?

Reply:
Yeah the spells were calling it but it was never inherited and the base method was empty. Plus `OnDamagedBySpell` exists as well which would serve the same purpose.